ui/src/selectors/index.js

```diff
@@ -44,15 +49,26 @@ const areColumnValuesEqual = (rows, index) => {
   });
 };
 
-const getSortIndex = (state, sortBy) => sortBy.index;
-const getSortDirection = (state, sortBy) => sortBy.direction;
+/**
+ * Gets the sort index from a sortBy object
+ * @param {Object} _state Redux state
+ * @param {Object} sortBy sortBy includes index and direction
+ */
+export const getSortIndex = (_state, sortBy) => sortBy.index;
+
+/**
+ * Gets the sort direction from a sortBy object
+ * @param {Object} _state Redux state
+ * @param {Object} sortBy sortBy includes index and direction
+ */
+export const getSortDirection = (_state, sortBy) => sortBy.direction;
 
 /**
  * Selector to convert app objects to array of app values
- *
- * @param {Array} apps - Array of app objects
+ * @param {Object} state Redux state
+ * @param {Object} _sortBy sortBy includes index and direction
  */
-export const getAppsTableRows = (state) => {
+export const getAppsTableRows = (state, _sortBy) => {
```

Comment:
Just curious - what rule(s) are we following to decide if a variable should be prefixed with `_` ?

Reply:
If it is not used, it is generally prefixed with an `_`

Reply:
@davidffrench got ya, so is `_sortBy` required at all as a parameter in this method?

Reply:
it is not required. I thought it would be good to list it though to show that all selector inputs get the same parameters